Problem with the Options type for VectorImage

**Describe the bug**

In #8996, the `imageRatio` property was added to the `Options` type in a way that is not supported by TypeScript.  See the output below.

**To Reproduce**

```
$ npm run typecheck
# ...

src/ol/layer/VectorImage.js:32:24 - error TS2339: Property 'imageRatio' does not exist on type 'Options'.

32     delete baseOptions.imageRatio;
                          ~~~~~~~~~~

src/ol/layer/VectorImage.js:39:32 - error TS2339: Property 'imageRatio' does not exist on type 'Options'.

39     this.imageRatio_ = options.imageRatio !== undefined ? options.imageRatio : 1;
                                  ~~~~~~~~~~

src/ol/layer/VectorImage.js:39:67 - error TS2339: Property 'imageRatio' does not exist on type 'Options'.

39     this.imageRatio_ = options.imageRatio !== undefined ? options.imageRatio : 1;
                                                                     ~~~~~~~~~~
```

**Expected behavior**

The `imageRatio` property should be type checked, but it is not.

TypeScript does support an intersection type expression for this sort of thing (see Microsoft/TypeScript#20077).  But JSDoc chokes on this syntax.

So this change would fix the type checking problem but break the documentation task:
```diff
diff --git a/src/ol/layer/VectorImage.js b/src/ol/layer/VectorImage.js
index 46d1b368d..45eb55fd3 100644
--- a/src/ol/layer/VectorImage.js
+++ b/src/ol/layer/VectorImage.js
@@ -6,11 +6,14 @@ import {assign} from '../obj.js';
 import CanvasVectorImageLayerRenderer from '../renderer/canvas/VectorImageLayer.js';
 
 /**
- * @typedef {import("./BaseVector.js").Options} Options
+ * @typedef {Object} ChildOptions
  * @property {number} [imageRatio=1] Ratio by which the rendered extent should be larger than the
  * viewport extent. A larger ratio avoids cut images during panning, but will cause a decrease in performance.
  */
 
+/**
+ * @typedef {import("./BaseVector.js").Options & ChildOptions} Options
+ */
 
 /**
  * @classdesc
```

We could create a new type for the vector image options, copying properties from the base vector options.  Or we could add `imageRatio` to the base vector options.  While it involves duplicating a lot of props, I think the first option is better.

Any other ideas?

--- a/src/ol/layer/VectorImage.js
+++ b/src/ol/layer/VectorImage.js
@@ -6,7 +6,41 @@ import {assign} from '../obj.js';
 import CanvasVectorImageLayerRenderer from '../renderer/canvas/VectorImageLayer.js';
 
 /**
- * @typedef {import("./BaseVector.js").Options} Options
+ * @typedef {Object} Options
+ * @property {number} [opacity=1] Opacity (0, 1).
+ * @property {boolean} [visible=true] Visibility.
+ * @property {import("../extent.js").Extent} [extent] The bounding extent for layer rendering.  The layer will not be
+ * rendered outside of this extent.
+ * @property {number} [zIndex] The z-index for layer rendering.  At rendering time, the layers
+ * will be ordered, first by Z-index and then by position. When `undefined`, a `zIndex` of 0 is assumed
+ * for layers that are added to the map's `layers` collection, or `Infinity` when the layer's `setMap()`
+ * method was used.
+ * @property {number} [minResolution] The minimum resolution (inclusive) at which this layer will be
+ * visible.
+ * @property {number} [maxResolution] The maximum resolution (exclusive) below which this layer will
+ * be visible.
+ * @property {import("../render.js").OrderFunction} [renderOrder] Render order. Function to be used when sorting
+ * features before rendering. By default features are drawn in the order that they are created. Use
+ * `null` to avoid the sort, but get an undefined draw order.
+ * @property {number} [renderBuffer=100] The buffer in pixels around the viewport extent used by the
+ * renderer when getting features from the vector source for the rendering or hit-detection.
+ * Recommended value: the size of the largest symbol, line width or label.
+ * @property {import("../source/Vector.js").default} [source] Source.
+ * @property {import("../PluggableMap.js").default} [map] Sets the layer as overlay on a map. The map will not manage
+ * this layer in its layers collection, and the layer will be rendered on top. This is useful for
+ * temporary layers. The standard way to add a layer to a map and have it managed by the map is to
+ * use {@link module:ol/Map#addLayer}.
+ * @property {boolean} [declutter=false] Declutter images and text. Decluttering is applied to all
+ * image and text styles, and the priority is defined by the z-index of the style. Lower z-index
+ * means higher priority.
+ * @property {import("../style/Style.js").StyleLike} [style] Layer style. See
+ * {@link module:ol/style} for default style which will be used if this is not defined.
+ * @property {boolean} [updateWhileAnimating=false] When set to `true`, feature batches will
+ * be recreated during animations. This means that no vectors will be shown clipped, but the
+ * setting will have a performance impact for large amounts of vector data. When set to `false`,
+ * batches will be recreated when no animation is active.
+ * @property {boolean} [updateWhileInteracting=false] When set to `true`, feature batches will
+ * be recreated during interactions. See also `updateWhileAnimating`.
  * @property {number} [imageRatio=1] Ratio by which the rendered extent should be larger than the
  * viewport extent. A larger ratio avoids cut images during panning, but will cause a decrease in performance.
  */

